Azure/sonic-buildimage
# [Mellanox] Update SDK/FW to 4.5.1208/2010.1218 and SAI version to 1.20.2.5

#### Why I did it

To include latest SDK fixes:

•	On CMIS modules, after low power configuration, the firmware waited for the module state to be ModuleReady instead of ModuleLowPower causing delays.
•	When connecting SN4600C, 100GbE port with CWDM4 module (Gen 3.0), link up  time is 30 seconds.


and to include SAI fixes \ changes:

* Reduce verbosity for resource check vendor data not found
* Fix metadata validation, check default value on conditions check
* Update SDK to 4.5.12XX, update port speeds
* Add 100MB, 10MB to 2201 system
* L3 VXLAN overlay ECMP
* VXLAN srcport API implementation
* Fix scheduler profile null (default values) when set on sub group scheduler group
* Fix ACL binding restoration when port leaves a LAG
* Fix route logic for set next hop/action and reference counter for ECMP overlay

#### How I did it

Updated SDK/FW submodule and relevant makefiles with the required versions.
Update SAI submodule and relevant makefile with the required version.

#### How to verify it

Build an image and run tests from "sonic-mgmt".

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
